### ISSUE TITLE
evaluating core stories with two stage fallback causes an error

### ISSUE THREAD

**Issue by Ghostvv** [RasaHQ/rasa_nlu#3187]
**Rasa version**: `master`

**Python version**: `3.6`

**Operating system** (windows, osx, ...): `mac`

**Issue**:
evaluating core stories with two stage fallback causes an error:
```
Traceback (most recent call last):
  File "/usr/local/Cellar/python/3.6.5_1/Frameworks/Python.framework/Versions/3.6/lib/python3.6/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/usr/local/Cellar/python/3.6.5_1/Frameworks/Python.framework/Versions/3.6/lib/python3.6/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/Users/ghost/Documents/rasa/rasa_core/rasa_core/evaluate.py", line 686, in <module>
    cmdline_arguments.e2e)
  File "/Users/ghost/Documents/rasa/rasa_core/rasa_core/evaluate.py", line 511, in run_story_evaluation
    use_e2e)
  File "/.../core/test.py", line 470, in collect_story_predictions
    in_training_data_fraction = _in_training_data_fraction(action_list)
  File "/...core/test.py", line 425, in _in_training_data_fraction
    a["action"] for a in action_list
  File "/.../core/test.py", line 426, in <listcomp>
    if not SimplePolicyEnsemble.is_not_memo_policy(a["policy"])
  File "/.../core/policies/ensemble.py", line 306, in is_not_memo_policy
    is_memo = best_policy_name.endswith(
AttributeError: 'NoneType' object has no attribute 'endswith'
```

**Comment by NomanSaleem4** [comment]
Getting this issue how can i resolve it . i am using

rasa==0.1.1
 rasa-addons==0.5.9
 rasa-core==0.14.4
 rasa-core-sdk==0.14.0
 rasa-nlu==0.15.0

**Comment by Ghostvv** [comment]
this should be fixed in rasa 1.0
